modules/ROOT/pages/dataweave-types-coercion.adoc

```diff
@@ -176,6 +176,46 @@ Using STRICT on an invalid date returns an error. The following example is an er
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 ----
 
+The following example converts two strings into date format and concatenates the result. The first string represents the date, the second string represents the time. The transformation uses `as` to coerce the first string to `LocalDateTime`, and then to a `String` with the desired format. The transformation also uses `++` to concatenate the result.
+
+.DataWeave script:
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/json
+var s1= 20201228 // (yyyymmdd),
+var s2= 1608 //(hhmm)
+---
+(s1 ++ s2)
+as LocalDateTime {format:"yyyyMMddHHmm"}
```

Comment:
We recommend using `uuuu` for dates because `yyyy` is for dates that involve A.D. (ACE)  and B.C. (BCE) eras. See https://docs.mulesoft.com/mule-runtime/4.3/dataweave-cookbook-format-dates#ex03
specifically, `//y is for use with the era (BCE or CE). Generally, use u, instead.
    "year-y" : myDateTime as String { format: "y"},`